.circleci/config.yml

```diff
@@ -56,3 +88,15 @@ workflows:
               ignore: /.*/
           requires:
             - build
+
+  build-arm64-compile-upload:
+    jobs:
+      - build-arm64
+      - upload-master:
+          filters:
+            branches:
+              only: master
+            tags:
+              ignore: /.*/
+          requires:
+            - build-arm64
```

Comment:
I don't see why the `build-compile-upload` workflow can't contain both jobs. Suggest adding the `build-arm64` job to that workflow instead of duplicating it.
```suggestion
```